cli/spec/kontena/cli/stacks/yaml/opto/vault_cert_prompt_resolver_spec.rb

```diff
@@ -1,4 +1,6 @@
 require 'opto'
+require 'kontena/cli/stacks'
+require 'kontena/cli/stacks/yaml/opto'
```

Comment:
Still failing somehow

```
An error occurred while loading ./spec/kontena/cli/stacks/yaml/opto/vault_cert_prompt_resolver_spec.rb.

Failure/Error: require 'kontena/cli/stacks'

LoadError:

  cannot load such file -- kontena/cli/stacks

# ./spec/kontena/cli/stacks/yaml/opto/vault_cert_prompt_resolver_spec.rb:2:in `require'

# ./spec/kontena/cli/stacks/yaml/opto/vault_cert_prompt_resolver_spec.rb:2:in `<top (required)>'
```

Reply:
Hmm. Works for me, bundled and unbundled.

Reply:
That is because I have an unstaged file in my working tree.